src/security/token/TKTokenSecretKey.h

```diff
@@ -16,10 +16,16 @@
 
 + (TKTokenSecretKey *)keyWithLevel:(Key_Level)level  privateKey:(NSData *)sk publicKey:(NSData *)pk;
 
++ (TKTokenSecretKey *)keyWithLevel:(Key_Level)level
+                        privateKey:(NSData *)sk
+                         publicKey:(NSData *)pk
+                        expiration:(long long) expiresAtMs;
+
 @property (nonatomic, readonly) NSString *id;
 @property (nonatomic, readonly) Key_Level level;
 @property (nonatomic, readonly) NSData *publicKey;
 @property (nonatomic, readonly) NSData *privateKey;
 @property (nonatomic, readonly) Key *keyInfo;
+@property (nonatomic, readonly) long long expiresAtMs;
```

Comment:
If we decide to use null instead of 0 represent "forever", we should use NSNumber instead of primitive type to avoid ambiguous.

IMO, Using 0 as "forever" might not be a bad idea considering consistency with the expiresAtMs in proto. Otherwise we should change the expiresAtMs in proto to string to avoid ambiguous in theory. It might be sth need to revisit later.